lib/reference/keywords.mjs

```diff
@@ -474,3 +474,6 @@ export const namedColorsKeywords = new Set([
 	'yellow',
 	'yellowgreen',
 ]);
+
+/** @see https://drafts.csswg.org/css-values/#typedef-calc-keyword */
+export const calcKeywords = new Set(['e', 'pi', 'infinity', '-infinity', 'NaN']);
```

Comment:
It might be better to treat any `ident` token as a valid calc keyword.

Specific features can define their own calc keywords and it will be more maintenance work to keep this list up to date. It will also add significant complexity to keep track of which keywords are valid in which context.

Given that this rule doesn't aim to validate the keywords used it might be better to consider any `ident` token a valid calc keyword.